e2e/testdata/fn-render/fnconfig-outside-package/Kptfile

```diff
@@ -0,0 +1,10 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app-with-db
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-label:unstable
+      configPath: labelconfig.yaml
+    - image: gcr.io/kpt-fn/set-label:unstable
+      configPath: some/../../home/.kube/config # config pointing outside the package
```

Comment:
do you restrict that function config must be in the *same directory* as the Kptfile?

Reply:
Discussed offline. We should restrict the fnconfig to be in the same package (but potentially a nested directory), but not in subpackages. Referring to wordpress package, it can be in:

`wordpress`
`wordpress/deployment`

but not any other directory.